client/app/components/land-use-action.js

```diff
@@ -2,120 +2,124 @@ import Component from '@glimmer/component';
-    },
-    {
-      name: 'Concession', field: 'dcpPfconcession', attr1: '', attr2: '',
-    },
-    {
-      name: 'Disposition of Real Property', field: 'dcpPfdispositionofrealproperty', attr1: '', attr2: '',
-    },
-    {
-      name: 'Franchise', field: 'dcpPffranchise', attr1: '', attr2: '',
-    },
-    {
-      name: 'Housing Plan & Project', field: 'dcpPfhousingplanandproject', attr1: '', attr2: '',
-    },
-    {
-      name: 'Landfill', field: 'dcpPflandfill', attr1: '', attr2: '',
-    },
-    {
-      name: 'Modification', field: 'dcpPfmodification', attr1: 'dcpPreviousulurpnumbers1', attr2: '',
-    },
-    {
-      name: 'Renewal', field: 'dcpPfrenewal', attr1: 'dcpPreviousulurpnumbers2', attr2: '',
-    },
-    {
-      name: 'Revocable Consent', field: 'dcpPfrevocableconsent', attr1: '', attr2: '',
-    },
-    {
-      name: 'Site Selection - Public Facility', field: 'dcpPfsiteselectionpublicfacility', attr1: '', attr2: '',
-    },
-    {
-      name: 'UDAAP', field: 'dcpPfudaap', attr1: '', attr2: '',
-    },
-    {
-      name: 'URA', field: 'dcpPfura', attr1: '', attr2: '',
-    },
-    {
-      name: 'Zoning Authorization', field: 'dcpPfzoningauthorization', attr1: 'dcpZoningauthorizationpursuantto', attr2: 'dcpZoningauthorizationtomodify',
-    },
-    {
-      name: 'Zoning Certification', field: 'dcpPfzoningcertification', attr1: 'dcpZoningpursuantto', attr2: 'dcpZoningtomodify',
-    },
-    {
-      name: 'Zoning Map Amendment', field: 'dcpPfzoningmapamendment', attr1: 'dcpExistingmapamend', attr2: 'dcpProposedmapamend',
-    },
-    {
-      name: 'Zoning Special Permit', field: 'dcpPfzoningspecialpermit', attr1: 'dcpZoningspecialpermitpursuantto', attr2: 'dcpZoningspecialpermittomodify',
-    },
-    {
-      name: 'Zoning Text Amendment', field: 'dcpPfzoningtextamendment', attr1: 'dcpAffectedzrnumber', attr2: 'dcpZoningresolutiontitle',
-    },
-  ];
-
   // Actions with partial or complete answers already saved
   // to the PAS Form Entity in CRM
   get existingLandUseActionSelections() {
     const { pasForm } = this.args;
-    return this.allLandUseActionOptions.filter((option) => pasForm[option.field] || pasForm[option.attr1] || pasForm[option.attr2]);
+    return allLandUseActionOptions.filter((option) => pasForm[option.countField] || pasForm[option.attr1] || pasForm[option.attr2]);
   }
 
-  // Actions selected by user in dropdown, or already
-  // existing in CRM
+  // Actions selected by user in dropdown ("new"), or already
+  // existing in CRM ("existing")
   get landUseActionSelections() {
+    const { pasForm } = this.args;
+    // Filter the "new" actions array so that the action does not exist on BOTH the "exisiting" and "new" arrays,
+    // when user enters a value in the action inputs (and therefore updates the model)
+    const filteredNewActions = this.newLandUseActionSelections.filter((action) => !pasForm[action.countField] && !pasForm[action.attr1] && !pasForm[action.attr2]);
```

Comment:
Ohh 💡  I see. So now filteredNewActions will hold the actions selected from the dropdown but not yet given any text inputs. A user might select multiple actions from the dropdown before scrolling through the action fields and filling them in.